docs/agents/llm-agents.md

```diff
@@ -399,7 +405,10 @@ Control whether the agent receives the prior conversation history.
 
 ### Code Execution
 
-![python_only](https://img.shields.io/badge/Currently_supported_in-Python-blue){ title="This feature is currently available for Python. Java support is planned/ coming soon."}
+<div class="language-support-tag" title="hover text.">
+   <span class="supported">Supported in ADK</span>
+   <span class="python">Python v0.1.0</span>
```

Comment:
I think the line break here is causing this badge to render with a space in between the boxes.

<img width="402" height="194" alt="Screenshot 2025-11-03 at 10 36 09 AM" src="https://github.com/user-attachments/assets/fe745ee7-a513-4b49-9105-e07cc3002090" />

Reply:
fixed.